sample-queries/sample-queries.json

```diff
@@ -270,7 +270,7 @@
         "humanName": "get a user's presence ",
         "requestUrl": "/beta/users/{user-id}/presence",
         "docLink": "https://docs.microsoft.com/en-us/graph/api/cloudcommunications-getpresencesbyuserid?view=graph-rest-beta&tabs=http",
-        "tip": "This query requires a user id and Presence.Read.All permissions. To find user id, you can run:  GET https://graph.microsoft.com/v1.0/users",
+        "tip": "This query requires a user id and Presence.Read.All permissions. To find a user id, you can run:  GET https://graph.microsoft.com/v1.0/users",
```

Comment:
Revise as: "This query requires a user ID and the Presence.Read.All permission. Use the following call to get a user ID:"